ktor-server/ktor-server-plugins/ktor-server-auth-jwt/jvm/src/io/ktor/server/auth/jwt/JWTAuth.kt

```diff
@@ -110,7 +108,8 @@ public abstract class JWTPayloadHolder(
      */
     public fun <T : Any> getListClaim(name: String, clazz: KClass<T>): List<T> {
         return try {
-            payload.getClaim(name).asList(clazz.javaObjectType)
+            val claim = payload.getClaim(name)
```

Comment:
can it be simplified to `payload.getClaim(name).asList(...) ?: null`?

Reply:
No, because `claim` is always not null.

Reply:
But `asList` returns `null`, right?